datafusion/expr/src/function.rs

```diff
@@ -37,10 +38,37 @@ pub type ScalarFunctionImplementation =
 pub type ReturnTypeFunction =
     Arc<dyn Fn(&[DataType]) -> Result<Arc<DataType>> + Send + Sync>;
 
+/// Arguments passed to create an accumulator
+pub struct AccumulatorArgs<'a> {
```

Comment:
I think we dont need to verify the `sort_exprs` is provided by user or not, because It is given only if `order by` expression is provided.